tdrs-backend/tdpservice/email/helpers/test/test_data_file.py

```diff
@@ -43,115 +43,115 @@ def test_send_data_submitted_email_no_email_for_pending(user, stt):
         (
             DataFile.Section.CLOSED_CASE_DATA,
             DataFileSummary.Status.ACCEPTED,
-            "Tribal Closed Case Data Processed Without Errors",
```

Comment:
this comment is for the whole PR not specific to this line:

I would suggest to add more tests around FRA datafile summary and the email being send for total number of errors. What if the case_aggregates doesn't have the "total_errors" field? are the "total_errors" being calculated correctly? I am just worried that we have misleading email being sent out with false 0 errors